docs/tutorials/index.md

```diff
@@ -30,20 +39,22 @@
 
 #### [Recommendation Systems](http://mxnet.io/tutorials/general_ml/recommendation_systems.html)
 
-## Python Tutorials
-### [NDArray](http://mxnet.io/tutorials/python/ndarray.html)
-### [Symbol](http://mxnet.io/tutorials/python/symbol.html)
-### [KVStore](http://mxnet.io/tutorials/python/kvstore.html)
-
-## R Tutorials
-### [NDArray](http://mxnet.io/tutorials/r/ndarray.html)
-### [Symbol](http://mxnet.io/tutorials/r/symbol.html)
-### [Write and use callback functions](http://mxnet.io/tutorials/r/CallbackFunctionTutorial.html)
-### [Neural Networks with MXNet in Five Minutes](http://mxnet.io/tutorials/r/fiveMinutesNeuralNetwork.html)
-### [Classify Real-World Images with Pre-trained Model](http://mxnet.io/tutorials/r/classifyRealImageWithPretrainedModel.html)
-### [Handwritten Digits Classification Competition](http://mxnet.io/tutorials/r/mnistCompetition.html)
-### [Character Language Model using RNN](http://mxnet.io/tutorials/r/charRnnModel.html)
-
-# Other Resources
+## Language Tutorials 
+
+### Python Tutorials
+#### [NDArray](http://mxnet.io/tutorials/python/ndarray.html)
```

Comment:
Would be better to replace `####` with `*` here as well and in the R tutorials.  Having every tutorial be a title makes the table of contents in the left-pane a huge mess.